### ISSUE TITLE
libunwind causes deadlock with heap-checker

### ISSUE THREAD

**Issue by alk** [gperftools/gperftools#225]
Originally reported on Google Code with ID 222

```
What steps will reproduce the problem?
In order to generate, you need my software, but i can't provide it.
I just did "pprof --gv p2.hprof.0001.heap"

What is the expected output? What do you see instead?
open the pprof.ps : arrows are all with zeros

What version of the product are you using? On what operating system?
google-perftools 1.5, libunwind 0.99 hardware : 64bits , ubuntu 9.10 up to 
date.

```

Reported by `laurent.debricon` on 2010-03-11 17:18:12

<hr>
- _Attachment: [p2.hprof.0001.heap](https://storage.googleapis.com/google-code-attachments/gperftools/issue-222/comment-0/p2.hprof.0001.heap)_

**Comment by dadler** [comment]
This issue is marked as Merged into #66 "Suggestion: Why are there only C++ interface in malloc_hook.h and malloc_extension.h ?"

That issue seems unrelated; should it actually be marked as merged into #69 "x86_64 trouble with libunwind"?  The same question applies other issues merged into #66.

**Comment by alk** [comment]
This is because when I migrated issues from code.google.com we had 3 issues open on github. So all tickets from code.google.com got ids offset by 3. So original issue #66 is in fact #69 on github. There might be some automated way to unbreak this mess.
